<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMonsterDropInfoItem</class>
 <widget class="QWidget" name="CMonsterDropInfoItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>84</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>怪物名称:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMonsterName">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>84</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>怪物代码:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMonsterID">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>84</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>掉落金钱:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxMoney">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>99999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>84</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>掉落药品几率:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxMedicanPercent">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>99999999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>84</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>掉落物品几率:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxItemPercent">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>99999999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>84</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>掉落物品数:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxCount">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>99999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
